src/CalculatorHelpers.ts

```diff
@@ -0,0 +1,533 @@
+          if (populationType) {
+            setResult(populationType, true, episodeResults.populationResults);
+
+            // set strata value
+          } else if (strataCode) {
+            if (episodeResults.stratifierResults) {
+              const strataResult = episodeResults.stratifierResults.find(strataResult => {
+                return strataResult.strataCode == strataCode;
+              });
+              if (strataResult) {
+                strataResult.result = true;
+              }
+            }
+          }
+
+          // else create a new episode using the list of all popcodes for the population
+        } else {
+          const newEpisodeResults: EpisodeResults = {
+            episodeId: episodeResource.id.value,
+            populationResults: []
+          };
+          populationGroup.population?.forEach(population => {
+            newEpisodeResults.populationResults.push({
+              populationType: <PopulationType>MeasureHelpers.codeableConceptToPopulationType(population.code),
+              result: false
+            });
+          });
+
+          if (populationGroup.stratifier) {
+            newEpisodeResults.stratifierResults = [];
+            let strataIndex = 1;
+            populationGroup.stratifier?.forEach(strata => {
+              const newStrataCode = strata.code?.text ?? `strata-${strataIndex++}`;
+              newEpisodeResults.stratifierResults?.push({
+                strataCode: newStrataCode,
+                result: newStrataCode == strataCode ? true : false
+              });
+            });
+          }
+
+          // Set the result for the current episode to true
+          if (populationType) {
+            setResult(populationType, true, newEpisodeResults.populationResults);
+          }
+
+          episodeResultsSet.push(newEpisodeResults);
+        }
+      }
+    });
+  }
+}
+
+/**
+ * Set all value set versions to 'undefined' so the execution engine does not grab the specified
+ * version in the ELM
+ *
+ * @param elm List of elm libraries to remove valueSet versions from.
+ * @returns The list passed in.
+ */
+export function setValueSetVersionsToUndefined(elm: ELM[]): ELM[] {
+  elm.forEach(elmLibrary => {
+    if (elmLibrary.library.valueSets != null) {
+      elmLibrary.library.valueSets.def.forEach(valueSet => {
+        if (valueSet.version != null) {
+          valueSet.version = undefined;
+        }
+      });
+    }
+  });
+  return elm;
+}
+
+/**
+ * Create the code service valueset database that the cql-execution engine needs.
+ *
+ * NOTE: This uses the `compose` attribue of the ValueSet to get code. This is incorrect and
+ * should be using the `expansion`. But current example measures have ValueSets with compose
+ * only.
+ *
+ * @param valueSetResources FHIR ValueSets.
+ * @returns The value set DB structure needed for the cql-execution CodeService.
+ */
+export function valueSetsForCodeService(valueSetResources: R4.IValueSet[]): any {
```

Comment:
Gonna guess this is my fault from grabbing stuff from Hoss's branch. Probably worth deleting one version or the other though.

Reply:
Yeah, I think this one should be deleted since the other one is more type-specific (returns the value set map instead of any), and lives in a standalone file, etc.

Reply:
I'll delete this. It's my bad, I told @okeefm he needed the function so I would remove my version of it.

Reply:
Deleted in bd92736